openquake/server/views.py

```diff
@@ -864,6 +866,15 @@ def aelo_validate(request):
     except Exception as exc:
         validation_errs[AELO_FORM_PLACEHOLDERS['siteid']] = str(exc)
         invalid_inputs.append('siteid')
+    try:
+        asce_version = request.POST.get('asce_version')
+        valid_asce_versions = \
+            oqvalidation.OqParam.asce_version.validator.choices
+        if asce_version not in valid_asce_versions:
+            raise ValueError(f"Valid ASCE versions are {valid_asce_versions}")
```

Comment:
See before, use the validator as it is meant to be used

Reply:
Done